doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -156,6 +156,11 @@ Please make sure to set your preferred storage location ***before*** migration.
   instead of `pimcore_core`. please add option `pimcore_asset_update` to command `bin/console messenger:consume pimcore_core... pimcore_asset_update` to post process assets on update.
   Also run command `bin/console messenger:consume pimcore_core` before the upgrade, so that `AssetUpdateTasksMessage` on the queue gets consumed.
 - [Events] Event `pimcore.element.note.postAdd` has been removed. Use `pimcore.note.postAdd` instead. Note: The event type changed from `ElementEvent` to `ModelEvent`.
+- Removed deprecated `SensioFrameworkExtraBundle` which affects the following:
+  - `@template` annotation must be replaced with `#[template]` attribute. Template guessing based on controller::action is not supported anymore.
```

Comment:
```suggestion
  - `@Template` annotation must be replaced with `#[Template]` attribute. Template guessing based on controller::action is not supported anymore.
```
Must be uppercase, to avoid confusion with the phpdoc one for `@template T`